### PR TITLE
MOL-53: Apple Pay Button after offcanvas change

### PR DESCRIPTION
**Pull request by Sironheart**

### CODE REVIEW

**Review by barbieswimcrew — APPROVED** [review]

File: Resources/views/frontend/checkout/ajax_cart.tpl [hunk 1]
```diff
@@ -7,6 +7,16 @@
             <div class="apple-pay--container apple-pay--container--ajax-cart">
                 {include 'frontend/plugins/payment/mollie_applepay_direct.tpl' }
             </div>
+            <script type="text/javascript">
+                // This needs to be inline to force the script on offcanvas reload
+                var applePayContainer = $('.apple-pay--container--ajax-cart');
```
Comment (boxblinkracer) [inline Resources/views/frontend/checkout/ajax_cart.tpl:12]: can we move this to javascript files please?

**Review by boxblinkracer — CHANGES_REQUESTED** [review]

File: Resources/views/frontend/checkout/ajax_cart.tpl (hunk 1 above)
Comment (Sironheart) [reply Resources/views/frontend/checkout/ajax_cart.tpl:12]: Sadly not, since there seems to be some issue with safari.

Comment (boxblinkracer) [reply Resources/views/frontend/checkout/ajax_cart.tpl:12]: ? all apple pay javascript code is placed in js files? that makes no sense?

Comment (Sironheart) [reply Resources/views/frontend/checkout/ajax_cart.tpl]: as discussed, we need to do the initApplePay functionality inline.

**Review by boxblinkracer — APPROVED** [review]

**Comment by boxblinkracer** [comment]
@Sironheart  can you fix the conflicts? then i would merge it, thanks

**Pull request merged**